student.rb

```diff
@@ -0,0 +1,11 @@
+class Student < Person
+  def initialize(age, classroom, name = nil, parent_permission: nil)
```

Comment:
- [ ] parent_permission by default should be true.
- [ ] name by default should be unknown

`def initialize(age, classroom, name = "Unknown", parent_permission = true)`